scaling-succotash.yaml

```
@@ -8,6 +8,9 @@ localhost:
 server:
   type: A
   value: 10.0.0.1
+telnet:
```

Telnet is bad mmkay, use SSH

```suggestion
ssh:
```